src/core/task/utils.ts

```diff
@@ -59,3 +60,79 @@ export const updateApiReqMsg = async (params: UpdateApiReqMsgParams) => {
 		} satisfies ClineApiReqInfo),
 	})
 }
+
+/**
+ * Common CLI tools that developers frequently use
+ */
+const CLI_TOOLS = [
```

Comment:
Very happy for you to edit this as you see fit to improve this, I don't have very strong feelings here - except maybe I'd quite like 'gh' and 'kubectl' to stay around :)